spec/index.html

```diff
@@ -1466,164 +1473,233 @@ <h2>Generalizations of RDF Triples, Graphs, and Datasets</h2>
+    <dd>It should be possible to transform any [=Full=] graph (resp. dataset) to a [=Classic=] graph (resp. dataset) using this method.
+      There is actually <a href="#section-classicize-caveat">a minor caveat</a> to this property.
     </dd>
   </dl>
 
+  <section id="section-classicize-definition">
+    <h2>From [=Full=] to [=Classic=]</h2>
 
-  <p>The general principle of the <a href="#section-unstar-algo">`unstar`</a> algorithm is to replace each <a>triple term</a> `S P O` in the input dataset with a fresh blank node.
-    This blank node is then use as the name of a new named graph containing a single triple `S P O`.
-    In order to distinguish those new named graphs from the original named graphs of the input dataset,
-    dedicated triples are added to a special `rdf:unstarMetadata` named graph.</p>
-
-  <p class=note>The blank nodes generated by `unstar` to replace <a>triple terms</a> should not be confused with the <a>reifiers</a> that are typically associated to these <a>triple terms</a>.</p>
+    <p>
+      Encoding an [=RDF graph=] to ensure that it is consumable by an RDF [=Classic=] implementation is called <dfn data-lt="classicize|classicized">classicizing</dfn> it.
+      It consists, while the graph has a [=triple term=] <var>tt</var> in its [=constituent terms=], in minting a fresh [=blank node=] <var>b</var>
+      (i.e. a blank node not in use in the graph), and replace <var>tt</var> with <var>b</var> in all the triples of the graph having <var>tt</var> in their [=constituents=].
+      Then the following triples are added to the graph (where <var>s</var>, <var>p</var> and <var>o</var> are respectively the [=subject=], [=predicate=] and [=object=] of <var>tt</var>):
+    </p>
+    <ul>
+      <li>(<var>b</var>, `rdf:type`, `rdf:TripleTerm`)
+      <li>(<var>b</var>, `rdf:ttSubject`, <var>s</var>)
+      <li>(<var>b</var>, `rdf:ttPredicate`, <var>p</var>)
+      <li>(<var>b</var>, `rdf:ttObject`, <var>o</var>)
+    </ul>
+    <p>This process is repeated until the graph has no [=triple term=] [=constituent=], and is therefore compliant with RDF [=Classic=].</p>
 
-  <p class=note>Despite having the same name, the `unstar` mapping defined in this specification is different from the `unstar` mapping defined in [[RDF-STAR-CG]].</p>
+    <p>Note that this transformation is <em>information preserving</em> only when the input graph does not contain at the same time a [=triple term=]
+      and an [=asserted=] triple (<var>b</var>, `rdf:type`, `rdf:TripleTerm`) where <var>b</var> is a [=blank node=].
+      Implementations encountering this situation MUST report an error.
+      See Section <a href="#section-classicize-caveat"></a> for a discussion on this limitation.
+    </p>
 
+    <p class=note>The blank nodes generated to replace [=triple terms=] should not be confused with the [=reifiers=] that are typically associated to these [=triple terms=].</p>
```

Comment:
It would be good to link this to the discussion of those "typical associations".
```suggestion
    <p class=note>The blank nodes generated to replace [=triple terms=] should not be confused with the [=reifiers=] that are typically associated with these [=triple terms=].</p>
```

Reply:
Good point about the link, will add it.